transport/control.go

```diff
@@ -149,45 +149,45 @@ type inFlow struct {
 	mu sync.Mutex
 	// pendingData is the overall data which have been received but not been
 	// consumed by applications.
-	pendingData uint32
```

Comment:
The http2 framer.WriteWindowUpdate(streamID, incr uint32) doesn't support int64. It's better to let the sizes be in uint32.